src/pages/tokenDrop/Landing.tsx

```diff
@@ -81,6 +81,19 @@ const Landing: FunctionComponent = () => {
         <Typography variant="subtitle1" sx={ { whiteSpace: "pre-wrap" } }>
           { projectDetails.description }
         </Typography>
+        <Typography variant="subtitle1" sx={ { mt: 2, whiteSpace: "pre-wrap" } }>
+          Explore the legal side of how Stream Tokens and royalties work in this
```

Comment:
@scandycuz The only thing that was different between the sales was the link where the "here" points to. I wanted to stay away from adding a third party library for now. If we need to change this in the future to use a string to html parser we can.